_posts/en/newsletters/2023-04-12-newsletter.md

```diff
@@ -0,0 +1,181 @@
+      pointed out that 110 GB is about 10% of his monthly ISP download
+      limit, so this is a significant reduction. Participants also expected
+      the percentage savings to increase with the recently expanded use of
+      witness data."
+  a1link="https://bitcoincore.reviews/27050#l-52"
+
+  q2="Would this improvement reduce the amount of download data from all
+      blocks back to the genesis block?"
+  a2="No, only since segwit activation (block height 481824); blocks before
+      segwit do not have witness data."
+  a2link="https://bitcoincore.reviews/27050#l-73"
+
+  q3="This PR implements two main changes, one to the block request logic
+      and one to block validation. What are these changes in more detail?"
+  a3="In the block request logic, remove `MSG_WITNESS_FLAG` from the fetch
+      flags so our peers don't send us the witness data. In validation,
+      skip witness merkle tree checks because we don't have the witness."
+  a3link="https://bitcoincore.reviews/27050#l-83"
+
+  q4="Without this PR, script validation is skipped under assume-valid,
+      but other checks that involve witness data are not skipped.
+      What are these checks that this PR will cause to be skipped?"
+  a4="The coinbase merkle root, the witness sizes, the maximum
+      witness stack items, and the block weight."
+  a4link="https://bitcoincore.reviews/27050#l-91"
+
+  q5="The PR does not include an explicit code change for skipping
+      all the extra checks mentioned in the previous question.
+      Why does that work out?"
+  a5="It turns out that all the extra checks just pass when you don't
+      have any witnesses.
+      This makes sense considering that segwit was a soft fork.
+      With the PR, we are essentially just pretending like we are a
+      pre-segwit node (up to the assume-valid point)."
+  a5link="https://bitcoincore.reviews/27050#l-117"
+%}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-alpha.0][] is a test release of the major changes to BDK
+  described in [Newsletter #243][news243 bdk].  Developers of
+  downstream projects are encouraged to begin integration testing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
+[Bitcoin Inquisition][bitcoin inquisition repo].*
+
+- [Core Lightning #6012][] implements several significant improvements
+  in Core Lightning's Python implementation that allow it to better work
```

Comment:
This reads like there is a whole second implementation of Core Lightning in Python?! TIL, if that’s the case. If it’s not the case, perhaps it should be clarified which parts of Core Lightning are Python or what this PR amends.